config/manifests/bases/rhods-operator.clusterserviceversion.yaml

```diff
@@ -42,10 +42,10 @@ metadata:
               "managementState": "Removed"
             },
             "kserve": {
-              "managementState": "Managed",
+              "managementState": "Managed"
```

Comment:
Was the CSV broken before this?

Reply:
It is the value of `operatorframework.io/initialization-resource` annotation, it should not be broken the csv itself but if it was used somewhere yes